scripts/load_inventory_demo.py

```
@@ -0,0 +1,387 @@
+    python scripts/load_inventory_demo.py --orders=30 -v
+    python scripts/load_inventory_demo.py --orders=20 --paid-ratio=0.5 -v
+    python scripts/load_inventory_demo.py --seed=1234 -v
+
+Requires:
+    Python 3.10+
+"""
+
+from argparse import ArgumentParser, Namespace
+from logging import ERROR
+from random import Random
+from sys import exit
```

Ruff (0.15.11)

[error] Import `exit` is shadowing a Python builtin

(A004)